Tests/SwiftParserTest/translated/EffectfulPropertiesTests.swift

```diff
@@ -188,12 +188,13 @@ final class EffectfulPropertiesTests: XCTestCase {
     assertParse(
       """
       var bad2 : Int {
-        get reasync { 0 }
+        get 1️⃣reasync { 0 }
         set reasync { }
       }
       """,
       diagnostics: [
         // TODO: Old parser expected error on line 2: expected '{' to start getter definition
+        DiagnosticSpec(message: "only function declarations may be marked 'reasync'; did you mean 'async'?")
```

Comment:
I think it is reasonable that we produce a similar diagnostics message here as ``rethrows``, since they're, in terms of syntax analysis, basically identical, although the message is not the same as the old parser. What do you think? @ahoppen

Reply:
Yes, I agree. Your diagnostic is way better than the old one. You can remove the TODO.